src/nurse/views.py

```diff
@@ -18,6 +22,64 @@
 )
 
 
+class SendEmailToDoctorView(APIView):
+    def post(self, request, pk):
+        prescription = get_object_or_404(Prescription, id=pk)
+
+        patient = prescription.patient
+        if not patient:
+            return Response(
+                {"error": "Patient not found"}, status=status.HTTP_404_NOT_FOUND
+            )
+
+        doctor_name = prescription.prescribing_doctor
+        email_doctor = prescription.email_doctor
+        if not email_doctor:
+            return Response(
+                {"error": "Doctor has no email"}, status=status.HTTP_400_BAD_REQUEST
+            )
+
+        user = request.user
+
+        if not user.first_name or not user.last_name:
+            return Response(
+                {"error": "Please update your profile with your first and last name"},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
```

Comment:
Unit test that branching and all the other branching we want full coverage

Reply:
It's done in the test